Symfony/CS/Console/Command/FixCommand.php

```diff
@@ -242,27 +259,14 @@ protected function configure()
 
 In combination with these config and command line options, you can choose various usage.
 
-For example, default level is ``psr2``, but if you also don't want to use
+For example, the default level is ``psr2``, but if you don't want to use
 the ``psr0`` fixer, you can specify the ``--fixers="-psr0"`` option.
 
 But if you use the ``--fixers`` option with only exact fixers,
 only those exact fixers are enabled whether or not level is set.
 
-With the <comment>--config-file</comment> option you can specify the path to the
-<comment>.php_cs</comment> file.
-
-By using ``--using-cache`` option you can set if caching
-mechanism should be used.
-
-Caching
--------
-
-The caching mechanism is enabled by default. This will speed up further runs by
-fixing only files that were modified. Tool will fix all files if tool version
-changed or fixers list changed.
-Cache is supported only for tool downloaded as phar file or installed via
-composer.
-Cache can be disabled via ``--using-cache`` option or config file:
+When using a configuration file the caching mechanism is enabled by default.
```

Comment:
You have splitted info about cache.
When you described `--using-cache` option and talk how cache works, and few § later tell it is enabled by default.

Now, out of the blue, you are telling it is enabled by default.
One more hint - even if you don't have any `.php_cs` file then standard `Symfony\CS\Config\Config` will be used -  some instance of configuration is always used, and the behavior of cache set by config class can be overriden by option. Not only when one has a config file.